### PR TITLE
KEYCLOAK-13289 Added redirect_url to Node.js logout

### PR DESCRIPTION
**Pull request by bender316**

### CODE REVIEW

File: securing_apps/topics/oidc/nodejs-adapter.adoc [hunk 1]
```diff
@@ -257,6 +257,9 @@ By default, the middleware catches calls to `/logout` to send the user through a
 configuration parameter to the `middleware()` call:
 [source,javascript]
     app.use( keycloak.middleware( { logout: '/logoff' } ));
+    
+When the user-triggered logout is invoked a query parameter `redirect_url` can be passed. 
+This parameter is then used as the redirect url of the OIDC logout endpoint.
```
Comment (abstractj) [inline securing_apps/topics/oidc/nodejs-adapter.adoc]: @bender316 thanks for submitting this. My suggestion is to not only include this paragraph, but also a snippet with examples. What do you think?

**Review by abstractj — CHANGES_REQUESTED** [review]
@bender316 thanks for submitting this. My suggestion is to not only include this paragraph, but also a snippet with examples. What do you think?

**Comment by bender316** [comment]
@abstractj I updated the section a bit with an example.

**Review by abstractj — APPROVED** [review]

**Comment by abstractj** [comment]
@bender316 thank you. I'm just waiting for Travis, but it looks good to me.

**Pull request merged**